microsoft/testsuites/performance/storageperf.py

```diff
@@ -187,6 +189,86 @@ def perf_storage_over_nfs_synthetic_tcp_4k(self, result: TestResult) -> None:
     def perf_storage_over_nfs_synthetic_udp_4k(self, result: TestResult) -> None:
         self._perf_nfs(result, protocol="udp")
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test data disk performance.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+    )
+    def perf_storage_generic_fio_test(
```

Comment:
I didn't see any data disk requirement set in case requirement.

Reply:
Yes, this is not specific to any disk type or network interface.